src/c++/vdo/base/slab-depot.c

```diff
@@ -5143,8 +4985,7 @@ void vdo_scrub_all_unrecovered_slabs(struct slab_depot *depot, struct vdo_comple
  *
```

Comment:
This is an unrelated thing I'd like you to roll in. Coud you fix the doc for get_block_allocator_statistics() to match upstream? That is, newline after "allocators" (not before), and align the second line correctly?

This is a place where somehow vdo-devel is out of step with our upstream baseline, I'd like to align them again.